src/hook.c

```diff
@@ -433,15 +448,26 @@ static VOID EnterSpinLock(VOID)
     SIZE_T spinCount = 0;
 
     // Wait until the flag is FALSE.
+#ifndef _MSC_VER
+    while (InterlockedCompareExchange(&g_isLocked, TRUE, FALSE) != FALSE)
+    {
+#else
     while (_InterlockedCompareExchange(&g_isLocked, TRUE, FALSE) != FALSE)
     {
         _ReadWriteBarrier();
-
+#endif
         // Prevent the loop from being too busy.
+#ifndef _MSC_VER
+        if (spinCount < 16)
+            usleep(250);
```

Comment:
I couldn't build this with MinGW/GCC.
```
../../src/hook.c: In function 'void EnterSpinLock()':
../../src/hook.c:462:23: error: 'usleep' was not declared in this scope
             usleep(250);
                       ^
../../src/hook.c:464:23: error: 'usleep' was not declared in this scope
             usleep(500);
                       ^
dllwrap: g++ exited with status 1
```
Maybe some additional installation will solve it. I don't think it deserves an effort.